### PR TITLE
Rel 5.6.35.v3

### PR DESCRIPTION
**Pull request by azhebel**

### CODE REVIEW

**Comment by kennt-percona** [comment]
lgtm

**Review by kennt-percona — APPROVED** [review]
lgtm

File: doc/source/release-notes/Percona-XtraDB-Cluster-5.6.35-26.20-3.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+* PXC-760 Fixed gcache page cleanup not triggering
+  when limits are exceeded.
+
+* :jirabug:`PXC-782`: Updated ``xtrabackup-v2`` script
+  to use the :option:`tmpdir` option
+  (if it is set under ``[sst]``, ``[xtrabackup]`` or ``[mysqld]``,
+  in that order).
+
+* :jirabug:`PXC-784`: Fixed the ``pc.recovery`` procedure to abort
```
Comment (kbauskar) [inline doc/source/release-notes/Percona-XtraDB-Cluster-5.6.35-26.20-3.rst:56]: This has external bug attached. Do we want to quite reference to it too.

Comment (azhebel) [reply doc/source/release-notes/Percona-XtraDB-Cluster-5.6.35-26.20-3.rst:56]: I can add link to LP bug as well. I did it for some RNs. But is it really needed? The link is available in the JIRA bug description.

Comment (kbauskar) [reply doc/source/release-notes/Percona-XtraDB-Cluster-5.6.35-26.20-3.rst:56]: Yes we should quote the lp bug.

**Comment by kbauskar** [comment]
LGTM.

**Review by kbauskar — APPROVED** [review]

**Pull request merged**